### PR TITLE
added cheat sheet images to vignettes

### PR DESCRIPTION
**Pull request by karima662**

### CODE REVIEW

File: vignettes/col_plan.Rmd [hunk 1]
```diff
@@ -295,4 +295,9 @@ Notice that the columns `mean` and `sd` are repeated for each of the treatment g
 
 In summary, the use of 1 or >1 `column` variables is a personal choice. Depending on the chosen workflow, a user can achieve the desired column ordering and labeling through a combination of data preparation and `tfrmt` options.
 
+## Help image/s
```
Comment (bzkrouse) [inline vignettes/col_plan.Rmd:298]:
```suggestion
## Help images
```

File: vignettes/body_plan.Rmd [hunk 2]
```diff
@@ -117,3 +117,8 @@ body_plan(
                                                                                  TRUE ~ frmt("x.xxx", missing = "")))
 )
 ```
+
+## Help image/s
+
+![tfrmt-frmts.jpg](images/tfrmt-frmts.jpg){width=100%}
```
Comment (bzkrouse) [inline vignettes/body_plan.Rmd:123]: When I render the Rmd this shows up as a caption for me - are you seeing the same? If so, I say we just remove it as such:

```{width=100%}```

File: vignettes/col_plan.Rmd [hunk 3]
```diff
@@ -295,4 +295,9 @@ Notice that the columns `mean` and `sd` are repeated for each of the treatment g
...
+
+![tfrmt-span_structure.jpg](images/tfrmt-span_structure.jpg){width=100%}
```
Comment (bzkrouse) [inline vignettes/col_plan.Rmd:300]: see comment above

File: vignettes/row_grp_plan.Rmd [hunk 4]
```diff
@@ -192,3 +186,17 @@ my_tfrmt <- tfrmt(
 
 print_mock_gt(my_tfrmt, .default = 1:2)
 ```
+
+## Help image/s
```
Comment (bzkrouse) [inline vignettes/row_grp_plan.Rmd:190]:
```suggestion
## Help images
```

File: vignettes/row_grp_plan.Rmd [hunk 5]
```diff
@@ -192,3 +186,17 @@ my_tfrmt <- tfrmt(
...
+
+<p align="center">
+
+![tfrmt-groups-three-cols-cropped.jpg](images/tfrmt-groups-three-cols-cropped.jpg){width=100%}
```
Comment (bzkrouse) [inline vignettes/row_grp_plan.Rmd:194]: see comment above

**Review by bzkrouse — CHANGES_REQUESTED** [review]

File: vignettes/body_plan.Rmd (hunk 2 above)
Comment (karima662) [reply vignettes/body_plan.Rmd]: resolved in latest commit

File: vignettes/col_plan.Rmd (hunk 3 above)
Comment (karima662) [reply vignettes/col_plan.Rmd]: resolved in latest commit

File: vignettes/row_grp_plan.Rmd (hunk 5 above)
Comment (karima662) [reply vignettes/row_grp_plan.Rmd]: resolved in latest commit

File: R/col_plan.R [hunk 6]
```diff
@@ -72,6 +72,13 @@
 #'    everything()
 #'    )
 #'
+#' @section Images:
+#' Here are some example outputs:
+#'
+#' \if{html}{\out{
+#' `r "<img src=\"https://raw.githubusercontent.com/GSK-Biostatistics/tfrmt/master/images/tfrmt-span_structure.jpg\" style=\"width:100\\%;\">"`
```
Comment (bzkrouse) [inline R/col_plan.R:79]:
```suggestion
#' `r "<img src=\"https://raw.githubusercontent.com/GSK-Biostatistics/tfrmt/main/images/tfrmt-span_structure.jpg\" style=\"width:100\\%;\">"`
```

File: R/tfrmt.R [hunk 7]
```diff
@@ -156,6 +156,14 @@
 #'   )
 #'
 #' )
+#'
+#' @section Images:
+#' Here are some example outputs:
+#'
+#' \if{html}{\out{
+#' `r "<img src=\"https://raw.githubusercontent.com/GSK-Biostatistics/tfrmt/master/images/tfrmt-groups-three-cols-cropped.jpg\" style=\"width:100\\%;\">"`
```
Comment (bzkrouse) [inline R/tfrmt.R:164]:
```suggestion
#' `r "<img src=\"https://raw.githubusercontent.com/GSK-Biostatistics/tfrmt/main/images/tfrmt-groups-three-cols-cropped.jpg\" style=\"width:100\\%;\">"`
```

File: R/theme_element.R [hunk 8]
```diff
@@ -32,6 +32,13 @@
 #'   )
 #' )
 #'
+#' @section Images:
+#' Here are some example outputs:
+#'
+#' \if{html}{\out{
+#' `r "<img src=\"https://raw.githubusercontent.com/GSK-Biostatistics/tfrmt/master/images/tfrmt-row_group_plan-cropped.jpg\" style=\"width:100\\%;\">"`
```
Comment (bzkrouse) [inline R/theme_element.R:39]:
```suggestion
#' `r "<img src=\"https://raw.githubusercontent.com/GSK-Biostatistics/tfrmt/main/images/tfrmt-row_group_plan-cropped.jpg\" style=\"width:100\\%;\">"`
```

**Review by bzkrouse — CHANGES_REQUESTED** [review]
This looks great! One small issue is that pkgdown requires images to be in /man/figures or /vignettes (building individual vignettes is fine, but `pkgdown::build_site()` reveals the issue). Can you please move the images to one of those, let's say /man/figures, and point to there instead?

**Comment by karima662** [comment]
@bzkrouse Shannon created a top level folder called images and placed png images into function documentation relative to this folder - so I followed this method. Also, Christina created an issue ticket to add this folder to Rbuildignore, so I'm assuming it should be okay to leave it as is?

**Comment by bzkrouse** [comment]
To recap, we should move/copy vignette related images to /man/figures and reference there. Also add man/figures to Rbuildignore. Hopefully everything is smooth but let me know!

**Review by bzkrouse — APPROVED** [review]

**Pull request merged**